app/imports/api/workingAgreements/server/publications.js

```diff
@@ -0,0 +1,14 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+//  import { isProjectMember } from '/imports/api/projects';
+//  import { isAdmin } from '/imports/api/users';
+import { WorkingAgreements } from './../WorkingAgreements.js';
+
+Meteor.publish('WorkingAgreements', function publishWorkingAgreements(sprintId) {
```

Comment:
This publications is not secure. Everyone who knows `sprintId` can get all working agreements. Make it more private!